phone_directory/settings.py

```diff
@@ -31,7 +31,7 @@
 # Application definition
 
 INSTALLED_APPS = [
-    'contacts',
+    'contacts.apps.MyDirectoryConfig',
```

Comment:
why did we need this change here?

Reply:
To use latest django and a better way as per django docs.